.github/workflows/build-packages.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches: [ master ]
   pull_request:
   workflow_dispatch:
+  schedule:
+    - cron:  '0 1 * * *'
```

Comment:
I don't know the cron syntax off the top of my head, so it would be good to include the extra comment about the frequency you are targeting here.

Reply:
Added